setup.cfg

```
@@ -9,7 +9,7 @@ ignore = F403,E226,E731,E275,W503,F405,E722,E741,W504,W605,E402
 exclude = .github
 
 [tool:pytest]
-python_files = *_test.py test_*.py *_example.py example_*.py
+python_files = tests/*test_*.py
```

```suggestion
python_files = tests/**/test_*.py
```

Doesn't something like this cover all potential depths of folder hierarchies?

---

My understanding is that this configuration file use a different syntax.
I tried exactly what you propose before, and it ended up only taking tests that have at least one subfolder. My understanding is that it consider `*` as a placeholder for any characters, *even* `/` (which is really weird, but that's my understanding.

---

Okay, weird stuff :sweat_smile: